test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -262,7 +263,11 @@ export default class BaseFormObject {
   }
 
   getErrorBoxText() {
-    return this.getErrorBox().innerText;
+    if (userVariables.gen3) {
+      return this.getErrorBox().find(CALLOUT_CONTENT_V3).innerText;
```

Comment:
Same idea as comment made in `CalloutObject.js`. I think we can remove the `CALLOUT_CONTENT_V3` selector and have this function and other relevant functions return all of the `innerText` in the alert component and then switch our spec assertions to use `.contains()` instead of `.eql()`

E.g. this function would become
```
if (userVariables.gen3) {
  return this.getErrorBox().innerText;
}
```

Reply:
Reverted